pulsar-broker/src/test/java/org/apache/pulsar/broker/admin/TopicPoliciesTest.java

```diff
@@ -3286,6 +3290,34 @@ public void testSetSubRateWithNoSub() throws Exception {
                 .build());
     }
 
+    /**
+     * Verify: {@link BacklogQuota#getPolicy()} can not be null.
+     */
+    @Test
+    public void testSetNonBacklogQuotType() throws Exception {
+        final NamespaceName ns = NamespaceName.get(myNamespace);
+        final String hostAndPort = pulsar.getWebServiceAddress();
+        final String nsPath = "/admin/v2/namespaces/" + ns + "/backlogQuota";
+        final String topicPath = "/admin/v2/persistent/" + ns + "/test-set-backlog-quota/backlogQuota";
+        admin.namespaces().setBacklogQuota(ns.toString(), BacklogQuota.builder().limitTime(1).limitSize(1).retentionPolicy(
+                BacklogQuota.RetentionPolicy.consumer_backlog_eviction).build());
+        BacklogQuota backlogQuotaWithNonPolicy = BacklogQuota.builder().limitTime(1).limitSize(1).build();
+        JerseyClient httpClient = JerseyClientBuilder.createClient();
```

Comment:
This leaks resources in the test. The `.close()` method of the `JerseyClient` instance should be called. Adding a Lombok `@Cleanup` annotation would handle it.

Reply:
> The test org.apache.pulsar.broker.admin.AdminApiTest.testRetentionAndBacklogQuotaCheck fails.

Fixed.

> This leaks resources in the test. The .close() method of the JerseyClient instance should be called. Adding a Lombok @Cleanup annotation would handle it.

Released